### PR TITLE
Initial load for Elastic APM telemetry component.

### PR DESCRIPTION
**Pull request by marcoshidalgonunes**
For reviewers: please confirm the component and its test are according to the Liquid Framework patterns.

### CODE REVIEW

File: src/Liquid.Core.Telemetry.ElasticApm/Extensions/DependencyInjection/IServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    /// <summary>
+    /// Extends <see cref="IServiceCollection"/> interface.
+    /// </summary>
+    public static class IServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Register telemetry interceptor <see cref="LiquidElasticApmInterceptor"/> and behaviour <see cref="LoggingBehaviour"/> for Elastic APM. 
+        /// </summary>
+        /// <param name="services">Extended <see cref="IServiceCollection"/> instance.</param>
+        public static IServiceCollection AddElasticApmTelemetry(this IServiceCollection services, IConfiguration configuration)
```
Comment (lucianareginalino) [inline src/Liquid.Core.Telemetry.ElasticApm/Extensions/DependencyInjection/IServiceCollectionExtensions.cs:20]: it is preferable that the injection method names have the prefix "AddLiquid".

File: src/Liquid.Core.Telemetry.ElasticApm/MediatR/LiquidPipelineBehavior.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+using Microsoft.Extensions.Logging;
+
+namespace Liquid.Core.Telemetry.ElasticApm.MediatR
+{
+    /// <summary>
+    /// Implements <see cref="IPipelineBehavior<TRequest, TResponse>"/> for Elastic APM.
+    /// </summary>
+    /// <typeparam name="TRequest">The type of request.</typeparam>
+    /// <typeparam name="TResponse">Type of response object obtained upon return of request.</typeparam>
+    public sealed class LiquidPipelineBehavior<TRequest, TResponse> : IPipelineBehavior<TRequest, TResponse>
```
Comment (lucianareginalino) [inline src/Liquid.Core.Telemetry.ElasticApm/MediatR/LiquidPipelineBehavior.cs:16]: The name of the class must represent what it is responsible for.
Example: "LiquidElasticApmTelemetryBehavior"

**Review by lucianareginalino — CHANGES_REQUESTED** [review]
The IPipelineBehavior implementation class must be in the project's "implementations" folder. it is not necessary to create a folder named "Mediator" as we are not implementing a component for this purpose.

**Comment by marcoshidalgonunes** [comment]
Requested changes are done. Waiting for confirmation there is all right.

**Comment by ycamargo** [comment]
We're approving this Pull Request. 
We will create the new pipeline to build, run tests and publish this new component to Nuget. 
We may ask for more changes if the build process shows any problem.

**Pull request merged**